ui/desktop/src/hooks/useRecipeManager.ts

```diff
@@ -101,6 +105,11 @@ export const useRecipeManager = (messages: Message[], locationState?: LocationSt
 
   // Pre-fill input with recipe prompt instead of auto-sending it
   const initialPrompt = useMemo(() => {
+    // Don't show initial prompt if we already have messages (ongoing conversation)
```

Comment:
are we calling this in the wrong place? the initial prompt shouldn't change because we have a conversation, we should just not put at as the non-initial prompt